Tools/CodeAnalyzerTestProject/Assets/Tests/CleanConsoleTest.cs

```diff
@@ -18,7 +18,11 @@ public class CleanConsoleTest
         public void TestCleanConsole()
         {
             string logs = Utilities.GetLogs();
-			Assert.That(logs, Is.Empty, string.Format("Found logs in the console on project startup.\n\nLogs: {0}",logs));
+            // This test is to pick up warnings generated by the Microsoft Roslyn analyzers. However, these may

```

Comment:
Which other packages are those? Maybe it would make more sense to make sure no other packages are in the project?

Reply:
Unfortunately, those warnings all came from the test framework -- which we can't get rid off..

Reply:
Ok, makes sense!